### PR TITLE
#167728005 Configure Eslint

### PR DESCRIPTION
**Pull request by Kingobaino**
What does this PR do?

 This PR helps to checkmate the codes that does not follow our standard guide

  Description of Task to be completed?

  Configured the Eslint to follow the following style guide
  - Airbnb Style guide
  - Import plugin
  - React plugin
 - And other set rules

  How should this be manually tested?

   This can be tested by including console.log in the code.

  What are the relevant stories?

  [#167728005](https://www.pivotaltracker.com/story/show/167728005)

  Screenshots (if appropriate)

  
Screenshot (5)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -93,4 +93,5 @@ typings/
 # DynamoDB Local files
 .dynamodb/
 
-# End of https://www.gitignore.io/api/node
\ No newline at end of file
+# End of https://www.gitignore.io/api/node
+package-lock.json
```
Comment (dieudonneAwa) [inline .gitignore:97]: Hey @Kingobaino, you should add an empty new line at the end of your file

File: .gitignore [hunk 2]
```diff
@@ -93,4 +93,5 @@ typings/
 # DynamoDB Local files
 .dynamodb/
 
-# End of https://www.gitignore.io/api/node
```
Comment (dieudonneAwa) [inline .gitignore:96]: A new line here too

**Comment by dieudonneAwa** [comment]
I think your screenshot should be that of an example code indicating Eslint errors when you pass your mouse over.

**Comment by dieudonneAwa** [comment]
Also, this PR is closed already.

**Review by biolabalo — APPROVED** [review]

File: .gitignore (hunk 2 above)
Comment (Abobos) [reply .gitignore:96]: Yes... new line

File: .gitignore (hunk 1 above)
Comment (Abobos) [reply .gitignore:97]: yes

**Pull request merged**